### PR TITLE
Use updated CaaS slurm appliance

### PR DESCRIPTION
**Pull request by sjpb**
- Uses the updated slurm app provided by https://github.com/stackhpc/caas-slurm-appliance/pull/10/.
- Uses the "fat" prebuild images produced by https://github.com/stackhpc/slurm_image_builder/

### CODE REVIEW

File: roles/azimuth/defaults/main.yml [hunk 1]
```diff
@@ -319,25 +319,27 @@ azimuth_caas_stackhpc_slurm_appliance_name: StackHPC Slurm Appliance
 # The git URL for the StackHPC Slurm appliance
 azimuth_caas_stackhpc_slurm_appliance_git_url: https://github.com/stackhpc/caas-slurm-appliance.git
 # The git version for the StackHPC Slurm appliance
-azimuth_caas_stackhpc_slurm_appliance_git_version: 3b115f03a673f876675813099531e75a4d9a13a7
+azimuth_caas_stackhpc_slurm_appliance_git_version: update/2022-9-29
```
[the remark this answers is not in the document]
Comment (sjpb) [reply roles/azimuth/defaults/main.yml:322]: This follows the key name in `azimuth_caas_stackhpc_slurm_appliance_project` (same file), but I'm happy to change if you prefer?

File: roles/community_images/defaults/main.yml [hunk 2]
```diff
@@ -62,9 +62,9 @@ community_images_default:
     source_url: https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_f0dc9cb312144d0aa44037c9149d2513/azimuth-images-prerelease/ubuntu-focal-desktop-220810-0904.qcow2
     source_disk_format: qcow2
     container_format: bare
-  rocky_8_20220702:
-    name: rocky-8-20220702
-    source_url: http://download.rockylinux.org/pub/rocky/8.6/images/Rocky-8-GenericCloud-8.6.20220702.0.x86_64.qcow2
+  openhpc_220830_2042:
+    name: openhpc-220830-2042
+    source_url: https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_3a06571936a0424bb40bc5c672c4ccb1/openhpc-images/openhpc-220830-2042.qcow2
```
[the remark this answers is not in the document]
Comment (sjpb) [reply roles/community_images/defaults/main.yml:67]: Not after merging the CaaS slurm app changes, no. This is the same image the main slurm app is currently using for CI.

File: roles/azimuth/defaults/main.yml [hunk 3]
```diff
@@ -319,25 +319,27 @@ azimuth_caas_stackhpc_slurm_appliance_name: StackHPC Slurm Appliance
...
 # The metadata root for the StackHPC Slurm appliance project
 azimuth_caas_stackhpc_slurm_appliance_metadata_root: >-
   https://raw.githubusercontent.com/stackhpc/caas-slurm-appliance/{{ azimuth_caas_stackhpc_slurm_appliance_git_version }}/ui-meta
 # The playbooks to use for the StackHPC Slurm appliance
 azimuth_caas_stackhpc_slurm_appliance_playbooks: [slurm-infra.yml]
+# Always update the cloned StackHPC Slurm appliance repo - useful for development on a branch
+azimuth_caas_stackhpc_slurm_appliance_alwaysUpdate: false
```
[the remark this answers is not in the document]
Comment (sjpb) [reply roles/azimuth/defaults/main.yml:329]: Its just following the key [here](https://github.com/stackhpc/ansible-collection-azimuth-ops/blob/a1758a6e9a6e7b863a2c876a11275146c490198f/roles/azimuth/defaults/main.yml#L378). Let me know if you still want to change it here to `azimuth_caas_stackhpc_slurm_appliance_always_update` though.

**Review by m-bull — APPROVED** [review]

**Review by mkjpryor — APPROVED** [review]
Working at JASMIN. LGTM.

**Pull request merged**